src/libs/actions/Report.js

```diff
@@ -500,9 +500,14 @@ function fetchChatReports() {
  * Get the actions of a report
  *
  * @param {Number} reportID
+ * @param {Number} offset
+ * @returns {Promise}
  */
-function fetchActions(reportID) {
-    API.Report_GetHistory({reportID})
+function fetchActions(reportID, offset) {
+    const reportActionsOffset = _.isNumber(offset)
```

Comment:
This is to account for the default of not passing an offset.

Reply:
Ok not sure if it matters that much, but maybe checking for `undefined` would be better since you want to account for a param that was not passed.

Reply:
Yeah for sure, but then we're not checking for `null` or `NaN` or whatever might come up - so I figure being specific about it being Numeric is good? Doesn't matter too much though so happy to change!

Reply:
Ah ok that makes sense. But I think maybe it does matter (at least a little). We are trying to paginate in the scenario but we accidentally sent `NaN` or `null`? Instead of alerting the developer we'll just cover the issue up by returning a response that is not at all what we would expect.

Maybe some combination of type check and undefined is best. Something like if we have an `offset` AND the type is valid then we use it, if we have an `offset` and the type is invalid we log an error, otherwise we have no `offset` at all so we assume the alternative logic is desired.